.rubocop_todo.yml

```diff
@@ -21,6 +21,16 @@ Style/OptionalBooleanParameter:
   Exclude:
     - 'lib/flagsmith.rb'
 
+Metrics/ClassLength:
```

Comment:
Seems like we're having to add a few exceptions here, as well as quite a few littered throughout the code. Are there any solutions to avoid this?

Reply:
I tried not to change the previous rules because the defaults are pretty good. On the other hand, there are places where following all the rules leads to over-engineering.